core/src/main/java/com/sequenceiq/cloudbreak/service/metrics/MetricService.java

```diff
@@ -86,6 +88,18 @@ public void submit(String metric, double value) {
         Metrics.gauge(metricName, gaugeCache, cache -> cache.getOrDefault(metricName, 0d));
     }
 
+    /**
+     * Set the specified tagged gauge value.
+     *
+     * @param metric Metric name
+     * @param value  Metric value
+     */
+    public void submit(String metric, String tag, double value) {
```

Comment:
Can we use "String tags..." as the last parameter and use the original submit method instead?

Reply:
Sure, with a minor difference. It has to be `Tag... tags`, because labels are name/value pairs, and in the original change I assumed the label name to be `"name"`.

Reply:
👍